airgun/entities/computeresource.py

```diff
@@ -94,7 +114,7 @@ def step(self, *args, **kwargs):
 
 @navigator.register(ComputeResourceEntity, 'Detail')
 class ComputeResourceDetail(NavigateStep):
-    VIEW = ResourceProviderDetailView
+    VIEW = ResourceProviderEditView
```

Comment:
Edit View for Details step? You're probably missing the point.
While most entities have only 1 "edit" page where entity values can be updated, some other entities have 2 separate screens. First screen can be accessed by clicking "Edit" button in "Actions" row, it allows to edit entity as usual, and second screen, which is accessed by clicking entity name in table. It won't allow you to edit anything, but show some entity details, logs, graphs etc. That's what we call 'Details' view. Example of such entity is hosts.
For entities with only 1 edit view naming isn't that big deal, but if compute resources have 2 separate screens - please use proper names and don't forget to implement separate views for each page.